src/lib/game.ts

```diff
@@ -212,22 +210,21 @@ export function Game (mainCanvas: HTMLCanvasElement, skier: Skier) {
     runningTime = 0
   }
 
-  this.step = (now: number) => {
-    if (paused) return
+  this.step = (lastStepAt: number, now: number) => {
+    if (paused) return;
```

Comment:
Sorry, but no `;` :joy: 

Feeling like a caveman in a codebase without formatter, having this kind of PR comment again.

For some reason everything was reformatted with [`standard`](https://standardjs.com/) [years ago](https://github.com/dividat/skifree.js/commit/ff53336e4634d8de164a48485b69fbe56a951d53).

Reply:
removed